frontend/src/templates/AvatarMenu/AvatarMenu.tsx

```diff
@@ -0,0 +1,45 @@
+import {
+  Box,
+  ButtonProps,
+  Icon,
+  MenuItemProps,
+  useMultiStyleConfig,
+} from '@chakra-ui/react'
+
+import { BxsUser } from '~/assets/icons/BxsUser'
+import Menu from '~/components/Menu'
+
+/**
+ * MenuButton styled for avatar
+ * Used to wrap Avatar component
+ * @preconditions Must be a child of Menu component,
+ * and returned using a render prop (see implementation in AvatarMenu.stories).
+ */
+export const AvatarMenuButton = (props: ButtonProps): JSX.Element => {
```

Comment:
since this file is an AvatarMenu, should we make this a subcomponent of the main AvatarMenu component (which is absent)?

There seems to warrant an AvatarMenu component which is basically the entire AvatarTemplate story in AvatarMenu.stories.tsx? 

Basically rule of thumb: the stories themselves should contain a "Default" story that is literally 

```tsx
const Template: Story = (args) => <Component {...args} />
```
which is literally rendering the component. If it requires more than that, the story is used incorrectly.

Reply:
thanks for advice! I extracted the AvatarMenu out, see if ok